src/orient/rational-calc.orient

```diff
@@ -0,0 +1,354 @@
+    kdf_hash_size = kdf_hash_size_gb * b_in_gb
+
+    window_size = window_size_mb * 1024 * 1024
+    windows = sector_size / window_size
+    window_nodes = nodes / windows
+
+  Time:
+    describe(porep_time, "Time to seal a sector (PoRep proof)", secs)
+    describe(porep_time_parallel, "Time to seal a sector (PoRep proof) with parallelization", secs)
+    describe(porep_proof_gen, "Time to generate a PoRep proof (SNARK, witness generation, loading data from storage medium)", secs)
+    describe(porep_proof_gen_parallel, "Time to generate a PoRep proof (SNARK, witness generation, loading data from storage medium) in parallel", secs)
+
+    porep_time = porep_proof_gen + porep_commit_time + encoding_time
+    porep_time_parallel = porep_proof_gen_parallel + porep_commit_time_parallel + encoding_time_parallel
+
+  Commitment:
+    describe(window_comm_tree_time, "Time to generate a merkle tree of stacked windows")
+    describe(window_comm_leaves_time, "Time to commit columns of a window")
+
+    describe(commit_size, "Size of a commitment (bytes)")
+    describe(commr_size, "Size of CommR (bytes)")
+    describe(commd_size, "Size of CommD (bytes)")
+    describe(commrlast_size, "Size of CommRLast (bytes)")
+
+    describe(commr_tree_depth, "Number of layers of the merkle tree with root hash CommR")
+
+    describe(seal_onchain_commitments_size, "Size of the seal commitments", bytes)
+    describe(porep_commit_time, "Time to generate CommC, CommR and CommQ", secs)
+    describe(porep_commit_time_mins, "Time to generate CommC, CommR and CommQ", mins)
+    describe(porep_commit_time_parallel, "Time to generate CommC, CommR and CommQ in parallel", secs)
+
+
+    describe(commr_time, "Time to generate CommR (seconds)")
+    describe(commd_time, "Time to generate CommD (seconds)")
+    describe(commc_time, "Time to generate CommC (seconds)")
+
+    commit_size = cid_size
+    // TODO: should be missing *2
+    window_comm_tree_time = merkle_tree_hash_time * window_nodes
+    window_comm_leaves_time = column_leaf_hash_time * windows
+    commr_size = commit_size
+    commd_size = commit_size
+    commrlast_size = commit_size
+
+    seal_onchain_commitments_size = commd_size + commr_size
+    // seal_commitment_time = commr_time
+
+    commd_time = merkle_tree_datahash_time * nodes
+
+    // TODO commr might be twice depending on CommQ
+    porep_commit_time = commr_time + commc_time
+    porep_commit_time_mins = porep_commit_time / 60
+
+    CommR (VectorR):
+      commr_time = merkle_tree_hash_time * nodes
+      commr_tree_depth = log2(nodes) 
+
+    CommR (ColumnR):
+      commr_time = window_comm_tree_time + window_comm_leaves_time 
+      commr_tree_depth = log2(window_nodes)
+
+    CommC:
+      describe(commc_tree_time, "Time to generate the merkle tree of CommC (after the leaves have been computed)", secs)
+      commc_tree_time = (window_comm_tree_time + window_comm_leaves_time) * drg_layers 
```

Comment:
Okay, I see that you're factoring in `drg_layers` here. It makes more sense to me to put this factor in the definition of `window_comm_leaves_time`, but I think this is equivalent.

Reply:
same as other comment